frontend/fingertips-frontend/lib/ViewsHelpers.ts

```diff
@@ -244,3 +255,9 @@ export async function getIndicatorData(
 
   return indicatorDataAllAreas;
 }
+
+export function determineBenchmarkRefType(lineChartAreaSelected?: string) {
```

Comment:
I think this one wants a return type hint:
```suggestion
export function determineBenchmarkRefType(lineChartAreaSelected?: string): BenchmarkReferenceType {
```